vortex-datafusion/src/persistent/opener.rs

```diff
@@ -81,7 +77,6 @@ impl FileOpener for VortexFileOpener {
 
         Ok(async move {
             let vxf = VortexOpenOptions::file(read_at)
-                .with_ctx(ctx.clone())
```

Comment:
The context is only used when a footer is not passed, so no point passing this all the way through since the footer cache handles this